boa/src/builtins/math/mod.rs

```diff
@@ -36,14 +36,14 @@ impl BuiltIn for Math {
 
         let attribute = Attribute::READONLY | Attribute::NON_ENUMERABLE | Attribute::PERMANENT;
         let object = ObjectInitializer::new(context)
-            .property("E", f64::consts::E, attribute)
-            .property("LN2", f64::consts::LN_2, attribute)
-            .property("LN10", f64::consts::LN_10, attribute)
-            .property("LOG2E", f64::consts::LOG2_E, attribute)
-            .property("LOG10E", f64::consts::LOG10_E, attribute)
+            .property("E", std::f64::consts::E, attribute)
+            .property("LN2", std::f64::consts::LN_2, attribute)
+            .property("LN10", std::f64::consts::LN_10, attribute)
+            .property("LOG2E", std::f64::consts::LOG2_E, attribute)
+            .property("LOG10E", std::f64::consts::LOG10_E, attribute)
             .property("SQRT1_2", 0.5_f64.sqrt(), attribute)
-            .property("SQRT2", f64::consts::SQRT_2, attribute)
-            .property("PI", f64::consts::PI, attribute)
+            .property("SQRT2", std::f64::consts::SQRT_2, attribute)
+            .property("PI", std::f64::consts::PI, attribute)
```

Comment:
Why this change? This is giving a bunch of warnings above due to unused imports.

Reply:
I didn't change anything here, it's just my fork getting old I guess

Reply:
No worries, a rebase should fix it, then.